sugarscape.py

```diff
@@ -893,50 +899,48 @@ def updateRuntimeStats(self):
-        self.runtimeStats["meanWealthHappiness"] = meanWealthHappiness
-        self.runtimeStats["meanHealthHappiness"] = meanHealthHappiness
-        self.runtimeStats["meanFamilyHappiness"] = meanFamilyHappiness
-        self.runtimeStats["meanSocialHappiness"] = meanSocialHappiness
-        self.runtimeStats["meanConflictHappiness"] = meanConflictHappiness
-        self.runtimeStats["meanTradePrice"] = meanTradePrice
-        self.runtimeStats["tradeVolume"] = tradeVolume
-        self.runtimeStats["giniCoefficient"] = self.updateGiniCoefficient() if len(self.agents) > 1 else 0
-        self.runtimeStats["meanAgeAtDeath"] = meanAgeAtDeath
-        self.runtimeStats["totalWealthLost"] += totalWealthLost
-        self.runtimeStats["totalMetabolismCost"] += totalMetabolismCost
-
-        self.runtimeStats["agentStarvationDeaths"] = agentStarvationDeaths
-        self.runtimeStats["agentDiseaseDeaths"] = agentDiseaseDeaths
-        self.runtimeStats["agentCombatDeaths"] = agentCombatDeaths
-        self.runtimeStats["agentAgingDeaths"] = agentAgingDeaths
-        self.runtimeStats["agentsReplaced"] = self.agentsReplaced
-        self.runtimeStats["agentsBorn"] = self.agentsBorn
-        self.agentsReplaced = 0
-        self.agentsBorn = 0
-        self.runtimeStats["agentWealthCollected"] = agentWealthCollected
-        self.runtimeStats["agentWealthTotal"] = agentWealthTotal
-        self.runtimeStats["agentWealthBurnRate"] = agentWealthBurnRate
-        self.runtimeStats["agentMeanTimeToLive"] = agentMeanTimeToLive
-        self.runtimeStats["environmentWealthCreated"] = environmentWealthCreated
-        self.runtimeStats["environmentWealthTotal"] = environmentWealthTotal
-
-        self.runtimeStats["agentTotalMetabolism"] = agentTotalMetabolism
-        self.runtimeStats["totalSickAgents"] = totalSickAgents
+        runtimeStats = {"agentAgingDeaths": agentAgingDeaths, "agentCombatDeaths": agentCombatDeaths, "agentDiseaseDeaths": agentDiseaseDeaths, "agentMeanTimeToLive": agentMeanTimeToLive,
+                        "agentStarvationDeaths": agentStarvationDeaths, "agentTotalMetabolism": agentTotalMetabolism, "agentWealthBurnRate": agentWealthBurnRate,
+                        "agentWealthCollected": agentWealthCollected, "agentWealthTotal": agentWealthTotal, "maxWealth": maxWealth, "meanAge": meanAge, "meanAgeAtDeath": meanAgeAtDeath,
+                        "meanConflictHappiness": meanConflictHappiness, "meanFamilyHappiness": meanFamilyHappiness, "meanHappiness": meanHappiness, "meanHealthHappiness": meanHealthHappiness,
+                        "meanMetabolism": meanMetabolism, "meanMovement": meanMovement, "meanSocialHappiness": meanSocialHappiness, "meanTradePrice": meanTradePrice, "meanWealth": meanWealth,
+                        "meanWealthHappiness": meanWealthHappiness, "meanVision": meanVision, "minWealth": minWealth, "population": numAgents, "sickAgents": sickAgents, "tradeVolume": tradeVolume
+                        }
```

Comment:
Do we get a stat for how many agents died in total for a given timestep? Not just the breakdown of the cause of death? I need a total death count for one of the graphs.

Reply:
Done.